com.toxicstoxm.LEDSuite.yml

```diff
@@ -0,0 +1,37 @@
+id: com.toxicstoxm.LEDSuite
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+
+finish-args:
+  - --share=network
```

Comment:
why does it need network?

Reply:
Network is needed to interface to the backend over WebSockets